panel/template/fast/fast.css

```diff
@@ -2,6 +2,10 @@
   --track-width: 2;
 }
 
+#header-design-provider fast-switch::part(checked-indicator) {
+  --neutral-foreground-rest: #181818;
```

Comment:
Doesn't this have to be set depending on the theme?

Reply:
I don't believe so. The "circle" in the swatch should always be black. It will always be shown on a white background. See pictures above.